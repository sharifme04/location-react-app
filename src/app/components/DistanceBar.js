import React from "react";
import { ReactComponent as CarSvg } from "../assets/icons/car-icon.svg";
import { ReactComponent as BusSvg } from "../assets/icons/bus-icon.svg";

const DistanceBar = () => {
  return (
    <div className="bar-container">
      <div className="distance-bar">
        <div className="car">
          <CarSvg />
          <p>24.1 km</p>
        </div>
        <div className="bus">
          <BusSvg />
          <p>24.8 km</p>
        </div>
      </div>
      <div className="triangle-container">
        <div className="triangle"></div>
      </div>
    </div>
  );
};

DistanceBar.propTypes = {};

export default DistanceBar;
