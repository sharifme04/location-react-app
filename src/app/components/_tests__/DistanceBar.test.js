import React from "react";
import renderer from "react-test-renderer";
import DistanceBar from "../DistanceBar";

test("snapshot Form renders with empty props", () => {
  const component = renderer.create(<DistanceBar />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});