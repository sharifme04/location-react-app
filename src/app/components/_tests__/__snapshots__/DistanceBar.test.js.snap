// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Form renders with empty props 1`] = `
<div
  className="bar-container"
>
  <div
    className="distance-bar"
  >
    <div
      className="car"
    >
      <svg>
        car-icon.svg
      </svg>
      <p>
        24.1 km
      </p>
    </div>
    <div
      className="bus"
    >
      <svg>
        bus-icon.svg
      </svg>
      <p>
        24.8 km
      </p>
    </div>
  </div>
  <div
    className="triangle-container"
  >
    <div
      className="triangle"
    />
  </div>
</div>
`;
